hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/balancer/ContainerBalancerTask.java

```diff
@@ -158,6 +158,14 @@ public ContainerBalancerTask(StorageContainerManager scm,
     this.selectedSources = new HashSet<>();
     this.selectedTargets = new HashSet<>();
     findSourceStrategy = new FindSourceGreedy(nodeManager);
+    if (config.getNetworkTopologyEnable()) {
+      findTargetStrategy = new FindTargetGreedyByNetworkTopology(
```

Comment:
I'd suggest to make field `findTargetStrategy` final

Reply:
Thanks for the review @Montura. In the interest of time, let's handle this in the next patch.